moveit_planners/pilz_industrial_motion_planner/test/integrationtest_command_planning.cpp

```diff
@@ -219,7 +219,7 @@ TEST_F(IntegrationTestCommandPlanning, PtpJointCart)
   Eigen::Isometry3d exp_iso3d_pose;
   tf2::fromMsg(expected_pose, exp_iso3d_pose);
 
-  EXPECT_TRUE(Eigen::Quaterniond(tf.rotation()).isApprox(Eigen::Quaterniond(exp_iso3d_pose.rotation()), EPSILON));
+  EXPECT_TRUE(Eigen::Quaterniond(tf.rotation()).isApprox(Eigen::Quaterniond(exp_iso3d_pose.rotation()), 2 * EPSILON));
```

Comment:
If a factor 2 solves the problem for you right now, I approve.

The tight requirement is still quite off the task though because the required plan above specifies a joint angle accuracy of 0.01rad for each absolute axis. So the maximum error we can get without violating the planning request is something way bigger (around 0.05 per array element assuming a unit rotation was requested)